docs/products/access/topics/custom-registration/custom-registration-v2.md

```diff
@@ -0,0 +1,145 @@
+# Custom Registration v1
+
+## Authentication
+
+This endpoint requires the client to provide the Client Authentication in the form of a PrivateKeyJWT 
```

Comment:
Please describe the conditions for the PrivateKeyJWT. It currently requires a specific algorithm.